Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -174,12 +195,20 @@ public Page GetPlatformEditPage()
                 {
                     return platformPage;
                 }
-            }          
+            }
             return null;
         }
 
         void ShowPlatformSpecificPage()
         {
+            //if widgets then no need to load platform specfic page
+            if (Convert.ToBoolean(mAction.GetInputParamValue(ActUIElement.Fields.WidgetsElement)) && mAction.Platform.Equals(ePlatformType.Java))
```

Comment:
this is java specific code and should not be here. 
Need to handle in the relevant platform

Reply:
pushed code for above all changes.